imports/plugins/included/connectors-shopify/lib/collections/schemas/shopifyConnect.js

```diff
@@ -35,6 +51,15 @@ const Webhook = new SimpleSchema({
 
 registerSchema("Webhook", Webhook);
 
+/**
+ * @name ShopifyConnectPackageConfig
+ * @type {SimpleSchema}
+ * @property {String} settings.apiKey Shopify API key
+ * @property {String} settings.password Shopify API password
+ * @property {String} settings.sharedSecret Shopify API shared secret
+ * @property {String} settings.shopName Shop slug
+ * @property {Array} settings.webhooks Array of registered Shopify webhooks
```

Comment:
For properties that are arrays of other schemas, is there a good way to link to the other schema from the description of this property?

Reply:
This is possible!

Looks like this in the code:
`* @property {Metafield[]} metafields optional`

Rendered:
![screen shot 2017-10-18 at 1 31 52 pm](https://user-images.githubusercontent.com/3673236/31741353-d2a4a7fe-b408-11e7-95a8-e02a630c9744.png)

And if `jsdoc` finds a `Metafield` symbol, it will link to the symbol accordingly.

So that line could be rewritten like:
`* @property {Webhooks[]} webhooks Array of registered Shopify webhooks`

and that will link to the Webhooks Schema

Reply:
The less automagical way to link from one symbol to another in the code is like this:
```
@see {@link foo} for further information.
```

http://usejsdoc.org/tags-see.html

Reply:
the automagical way is pretty cool